src/python/pants/goal/run_tracker.py

```diff
@@ -387,15 +390,51 @@ def store_stats(self):
     if target_data:
       run_information['target_data'] = ast.literal_eval(target_data)
 
+    def get_option(scope_or_scope_with_flag, flag=None):
+      """Looks up a scope in the options parsed by Pants.
+
+      scope_or_scope_with_flag is a string either either of form scope or scope.flag
+      If it's a scope, the scope's values will be gotten as a dict, and either returned if flag is
+        None, or else the flag will be looked up inside the dict.
```

Comment:
This seems overcomplicated. Why do we need two different ways of picking an option? Is it that important to be able to cherrypick single options out of a scope at all?

Reply:
Yeah, that's fair - the ^ delimiter makes it a lot simpler.

Some scopes are pretty large, so it's handy to be able to narrow; I think the interface is now simple enough, but happy to discuss if you disagree